packages/in_app_purchase/in_app_purchase_storekit/ios

```diff
@@ -0,0 +1 @@
+shared
```

Comment:
I missed this before, unfortunately this won't work due to https://github.com/dart-lang/pub/issues/3143. You'll need to symlink each file rather than the directories. (I filed https://github.com/flutter/flutter/issues/115085 since I think this is a clear case where that's a pretty unfortunate requirement, but we should proceed with doing it the way that works now, and hopefully be able to someday come back and simplify the plugin.)